lib/cdo/rack/attack.rb

```diff
@@ -0,0 +1,59 @@
+require 'rack/attack'
+require 'active_support/core_ext/numeric/time'
+
+class Rack::Attack
+  # maximum requests per minute over a 4-minute stretch.
+  MAX_TABLE_READS_PER_MIN = 2 * 60
```

Comment:
My understanding is that our top priority right now is to get this live. Perhaps our best approach is to ship this with these as constants, and then move them to DCDO in a subsequent PR? Unless putting them in DCDO is entirely trivial...